.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build, test, lint
+      run: |
+        npm install
+        npm run build
+        npm test
```

Comment:
Can we parallelize `lint` and `test`?

Reply:
Sort-of yes.  You can define dependencies between jobs, and then we could have separate `build`, `test` and `lint` jobs, and use the [download artifact](https://github.com/actions/download-artifact) action to transfer data between them (it also uploads).
Not doing it here, I don't think it will end up saving any time.